react/src/components/VFolderPermissionCell.tsx

```diff
@@ -33,19 +35,24 @@ const VFolderPermissionCell: React.FC<VFolderPermissionCellProps> = ({
     },
   };
 
-  const vfolder = useFragment(
+  const vfolderData = useFragment(
     graphql`
       fragment VFolderPermissionCellFragment on VirtualFolderNode {
         permissions
       }
     `,
-    vfolderFrgmt,
+    vfolderFrgmt ?? null,
   );
 
-  const permission = _.includes(vfolder?.permissions, 'mount_rw') ? 'rw' : 'ro';
+  const permission = useMemo(() => {
```

Comment:
Caching the result of permissionMap[permission] would enhance performance for various use cases of this component.